test/integration/targets/gathering_facts/test_gathering_facts.yml

```diff
@@ -129,3 +129,48 @@
           - 'ansible_interfaces|default("UNDEF_NET") != "UNDEF_NET"'
           - 'ansible_mounts|default("UNDEF_HW") == "UNDEF_HW"'
           - 'ansible_virtualization_role|default("UNDEF_VIRT") == "UNDEF_VIRT"'
+
+- hosts: facthost9
```

Comment:
There is no `facthost9`, so these tests are not actually running, as seen in the CI logs here:

https://app.shippable.com/runs/585c33c6666b4b1000d0575d/33/console